EmployeeAPI/Controllers/DeveloperController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Employees.Models;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EmployeeAPI.Controllers
+{
+    /// <summary>
+    ///  Developers Controller
+    /// </summary>
+    [Route("api/v1/developer")]
+    [Produces("application/json")]
+    [ApiController]
+    public class DeveloperController : ControllerBase
+    {
+        /// <summary>
+        /// Calculate Developers salary : double
+        /// </summary>
+        /// <param name="salary">: double</param>
+        /// <returns>salary value</returns>
+        [HttpGet("calculate-salary")]
+        public double CalculateSalary([FromQuery] double salary)
+        {
+            var dev = new Developer("Noor", salary);
```

Comment:
create class for all the data to be as database. and this method should just pass the employee id from client